plugin/path_role.go

```diff
@@ -7,7 +7,9 @@ import (
 	"strings"
 	"time"
 
+	"github.com/fatih/structs"
```

Comment:
We're trying to avoid using third-party libraries that belong to one person like this as much as possible. Also, I think @jefferai mentioned Vault was moving specifically away from using structs for converting the response and just writing it out instead. 

Otherwise, this requires updates to deps